pyitau/main.py

```diff
@@ -87,6 +91,71 @@ def get_statements_from_month(self, month=1, year=2001):
         )
         return response.json()
 
+    def get_pix(self, days=90):
+        """Get and return the pix transaction for the last {days}."""
+
+        response = self._session.post(
+            ROUTER_URL, headers={"op": self._bigger_menu_page.pix_statements_op})
+        self.pix_page = PixPage(response.text)
+
+        data = {
+            "periodo": f"{days}",
+            "page": "1",
+            "filtro": "debito",
+            "ordenacao": "desc",
+        }
+
+        headers = {
+            "Op": self.pix_page.pix_statements_op,
+            "X-FLOW-ID": self._flow_id,
+            "X-CLIENT-ID": self._client_id,
+            "X-Requested-With": "XMLHttpRequest",
+            "Accept": "application/json, text/javascript, */*; q=0.01",
+            "Content-Type": "application/x-www-form-urlencoded; charset=UTF-8",
+        }
+
+        response = self._session.post(ROUTER_URL, data=data, headers=headers)
+        return response.json()['lancamentos']
+
+    def get_pix_receipts(self, pix_transactions):
+        """Returns a generator containing the binary files of the given pix transactions."""
+
+        headers = {
+            "Accept": "*/*",
+            "Accept-Encoding": "gzip",
+            "Content-type": "application/x-www-form-urlencoded",
+            "Op": self.pix_page.pix_impressao_op,
+        }
+
+        path = os.path.abspath(Path(__file__) / ".." / "files" / "pix_pdf.html")
```

Comment:
**Suggestion:** Avoid using os.path for file path manipulations. Since you are already using pathlib, stick to it for consistency.
```suggestion
        path = Path(__file__).parent / "files" / "pix_pdf.html"
```